redisraft.h

```diff
@@ -485,7 +486,9 @@ typedef struct ShardingInfo {
      * since a zero value indicates the slot is unassigned. The index
      * should therefore be adjusted before refering the array.
      */
-    ShardGroup *hash_slots_map[REDIS_RAFT_HASH_SLOTS];
+    ShardGroup *stable_slots_map[REDIS_RAFT_HASH_SLOTS];
```

Comment:
What is the rationale behind storing three maps vs. a single map and hold the state in the ShardGroup?

Reply:
because in a migrating case, one will have migrating and importing shard groups for the same slot.